services/core-web/src/components/search/SearchBarDropdown.js

```diff
@@ -12,78 +12,86 @@ const propTypes = {
   searchBarResults: PropTypes.arrayOf(PropTypes.any).isRequired,
 };
 
-const staticMenuItems = [
-  <Menu.ItemGroup title="Lookup">
-    <Menu.Item key="/dashboard/mines">
+export const SearchBarDropdown = (props) => {
+  const createStaticMenuItem = (text, url, img) => (
+    <Menu.Item key={url}>
       <p>
-        <img
-          alt="Mine"
-          className="padding-small--right vertical-align-sm icon-svg-filter"
-          height="25px"
-          src={MINE}
-        />{" "}
-        Mines
+        <img className="icon-svg-filter" src={img} alt={text} height={25} />
+        {text}
       </p>
     </Menu.Item>
-    ,
-    <Menu.Item key="/dashboard/contacts">
-      <p>
-        <img
-          alt="team"
-          className="padding-small--right vertical-align-sm icon-svg-filter"
-          height="25px"
-          src={TEAM}
-        />{" "}
-        Contacts
-      </p>
-    </Menu.Item>
-  </Menu.ItemGroup>,
-];
+  );
+
+  const staticMenuItems = [
+    <Menu.ItemGroup title="Look up">
+      {createStaticMenuItem("Mines", route.MINE_HOME_PAGE.route, MINE)}
+      {createStaticMenuItem("Contacts", route.CONTACT_HOME_PAGE.route, TEAM)}
+    </Menu.ItemGroup>,
+  ];
 
-const URLFor = (item) =>
-  ({
-    mine: route.MINE_GENERAL.dynamicRoute(item.result.id),
-    party: route.PARTY_PROFILE.dynamicRoute(item.result.id),
-    permit: route.MINE_PERMITS.dynamicRoute(item.result.id),
-  }[item.type]);
+  const URLFor = (item) =>
+    ({
+      mine: route.MINE_GENERAL.dynamicRoute(item.result.id),
+      party: route.PARTY_PROFILE.dynamicRoute(item.result.id),
+      permit: route.MINE_PERMITS.dynamicRoute(item.result.id),
```

Comment:
Do we actually search through permits? If so, I'll make a section for those to go with Mines and Contacts.